src/script/conversation/MessageRepository.ts

```diff
@@ -138,16 +137,23 @@ type ClientMismatchHandlerFn = (
   consentType?: CONSENT_TYPE,
 ) => Promise<boolean>;
 
+/** A Quote that is meant to be sent in a message (and thus needs to have a valid hash) */
+type OutgoingQuote = QuoteEntity & {hash: Uint8Array};
```

Comment:
wondering if this should be exported and reused since it appears again InputBarViewModel

Reply:
Yep, that's good call :+1: